app/content/newsletter-automatic.md

```diff
@@ -19,7 +19,7 @@ const newsletterCache = new NewsletterPersistence();
 // Sample newsletter configuration created by the publisher
 const publisherConfiguration = {
   name: 'Subscriber Newsletter',
-  configurationId: '49c12712-9750-4571-8c67-96722561c13a',
```

Comment:
It belongs to **RRM:E - Newsletter Demo**  not to **Reader Revenue: Enterprise Demo**